### DISCUSSION TITLE
Docker Container Password

### DISCUSSION THREAD

**Discussion by RonTek567** [tteck/Proxmox#1112] [Help Needed] [answered]
Hello, i have created a new docker container. I need the login name and password for the shell.

**Comment by tteck** [comment, marked as answer]
If you didn't set a root password and it's asking for credentials, the script didn't complete successfully. Stay on the shell until the script finishes.

**Reply by RonTek567** [reply]
thank you very much. previously i used the standard settings an i think there was no question for the password.
i maked a new container with advanced settings and choose a password.
now it works

**Comment by deepbrainx** [comment]
Related question. Once the script is done I just see a prompt in the console. What are the ports I'm supposed to access?
